specs/merge/beacon-chain.md

```diff
@@ -181,7 +178,7 @@ class ExecutionPayloadHeader(Container):
     gas_limit: uint64
     gas_used: uint64
     timestamp: uint64
-    base_fee_per_gas: uint64
+    base_fee_per_gas: Bytes32
```

Comment:
Little endian or big endian?

Reply:
This is a good question. For better compatibility with the future Consensus API, it probably should be big-endian, it would allow to avoid reversing bits each time when execution payload is either sent to or received from execution client. But it might be controversial to the beacon chain spec which is little-endian in general. What are your thoughts?

Reply:
But if we then convert this field to `uint256` using little-ending from now should make it forward-compatible.

Reply:
Both have pros/cons, I am ok with either, let's just make sure to spec how to interpret this 32 bytes.

Reply:
>Little endian or big endian?

Seems it's chosen big endian here. If you decide to go with little endian remember to change back `GENESIS_BASE_FEE_PER_GAS`